### DISCUSSION TITLE
How to close all other tabs?

### DISCUSSION THREAD

**Discussion by salmankhilji** [kovidgoyal/kitty#4942] [Q&A] [answered]
I am currently using the following to close all tabs but the focused one:

```sh
for id in $(kitty @ ls | jq '.[0].tabs | .[] | select(.is_focused == false) | .id'); do kitty @ close-tab --match=id:$id; done
```

Is there a better way to do this?

**Comment by page-down** [comment, marked as answer]
If you need to use it in kitty, for example to bind shortcuts, then writing a kitten implementation is the best way to go.
There are many tab manager related ones in the code for reference.

**Reply by salmankhilji** [reply]
Available in `8ae2f068`.  Thank you!
